demos/python_demos/smart_library_demo/smart_library_demo.py

```diff
@@ -0,0 +1,252 @@
+                        Available DNN face recognizer - DNNfr')
+    parser.add_argument('-m_rd', type = str, 
+                        dest = 'rdModel', required=True,  
+                        help = 'Required. Path to .xml file')        
+    parser.add_argument('-fd', type = str, 
+                        dest = 'fdDet', required=True, 
+                        help = 'Required. \
+                        Type of detector. \
+                        Available DNN face detector - DNNfd')          
+    parser.add_argument('-m_fd', type = str, 
+                        dest = 'fdModel', required=True, 
+                        help = 'Required. Path to .xml file')
+    parser.add_argument('-lm', type = str, dest = 'lmDet', required=True,
+                        help = 'Required. \
+                        Type of detector. \
+                        Available DNN landmarks regression - DNNlm')
+    parser.add_argument('-m_lm', type = str, 
+                        dest = 'lmModel', required=True, 
+                        help = 'Required. \
+                        Path to .xml file')    
+    parser.add_argument('-w_rd', type = int, default = 128,
+                        dest = 'rdWidth',
+                        help = 'Optional. Image width to resize')
+    parser.add_argument('-h_rd', type = int, default = 128,
+                        dest = 'rdHeight',
+                        help = 'Optional. Image height to resize') 
+    parser.add_argument('-t_rd', type = float, default = 0.8,
+                        dest = 'rdThreshold',
+                        help = 'Optional. \
+                         Probability threshold for face detections.')
+    parser.add_argument('-w_fd', type = int, default = 300,
+                        dest = 'fdWidth',
+                        help = 'Optional. Image width to resize')
+    parser.add_argument('-h_fd', type = int, default = 300,
+                        dest = 'fdHeight',
+                        help = 'Optional. Image height to resize') 
+    parser.add_argument('-t_fd', type = float, default = 0.9,
+                        dest = 'fdThreshold',
+                        help = 'Optional. \
+                        Probability threshold for face detections.')
+    parser.add_argument('-w_lm', type = int, default = 48,
+                        dest = 'lmWidth',
+                        help = 'Optional. Image width to resize')
+    parser.add_argument('-h_lm', type = int, default = 48,
+                        dest = 'lmHeight',
+                        help = 'Optional. Image height to resize') 
+    parser.add_argument('-br', type = str,  default = 'QR',
+                        dest = 'br',
+                        help = 'Optional. Type - QR')
+    parser.add_argument('-lib', type = str, default = 'library.json',
+                        dest = 'lib',
+                        help = 'Optional. Path to library.') 
+    parser.add_argument('-w', type = int, default = 0,
```

Comment:
The name of the key is usually `'-i', '--input'`, it usually accepts a string, and tries casting it to int before passing it to VideoCapture, to support other sources.